packages/http/src/mocker/generator/JSONSchema.ts

```diff
@@ -9,12 +14,41 @@ jsf.option({
   optionalsProbability: 1,
   fixedProbabilities: true,
   ignoreMissingRefs: true,
-  useExamplesValue: true,
-  useDefaultValue: true,
   maxItems: 20,
   maxLength: 100,
 });
 
-export async function generate(source: unknown): Promise<unknown> {
-  return jsf.resolve(cloneDeep(source));
+export function generate(source: JSONSchema): unknown {
+  return jsf.generate(cloneDeep(source));
+}
+
+export function generateStatic(source: JSONSchema): unknown {
+  const transformedSchema = toOpenAPIJSONSchemaesque(source);
+  return sampler.sample(transformedSchema);
+}
+
+function hasExamples(source: JSONSchema): source is JSONSchema6 | JSONSchema7 {
+  return 'examples' in source;
+}
+
+function toOpenAPIJSONSchemaesque(schema: JSONSchema): any {
+  const returnedSchema = cloneDeep(schema);
+
+  ['properties', 'anyOf', 'allOf', 'oneOf'].forEach(property => {
+    if (!schema[property]) return;
```

Comment:
Maybe for consistency
```suggestion
    if (!returnedSchema[property]) return;
```